.github/workflows/test.yml

```diff
@@ -144,7 +144,7 @@ jobs:
         run: echo "RUST=$(rustc --version)" >> "$GITHUB_ENV"
 
       - name: Upload to codecov.io
-        uses: codecov/codecov-action@v5.0.4
+        uses: codecov/codecov-action@v5.0.7
```

Comment:
```suggestion
        uses: codecov/codecov-action@v5
```
It seems dependabot bugged out, as [the last release is 5.0.5](https://github.com/codecov/codecov-action/releases), and not 5.0.7.

I think we ought to revert this back to just the `v5` tag, as it [always points to the latest commit](https://github.com/codecov/codecov-action/releases/tag/v5) (at least it should), so we don't get these dependabot PRs every single day.

Reply:
Strange. I see `5.0.7` as a the least one
<img width="908" alt="image" src="https://github.com/user-attachments/assets/89fc07c5-151a-4f98-a787-7ecad8e113b7">

Reply:
Looking at the timestamps, it seems they released versions 5.0.6/7 a couple of hours after I posted my comment

Reply:
@qalisander @bidzyyys can we merge the [suggested change](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/415#discussion_r1851463935)?